spotbugs/src/main/java/edu/umd/cs/findbugs/ba/NullnessAnnotation.java

```diff
@@ -40,6 +40,8 @@ boolean match(@DottedClassName String className) {
                     || "com.google.common.base.Nullable".equals(className)
                     || "org.eclipse.jdt.annotation.Nullable".equals(className)
                     || "org.jetbrains.annotations.Nullable".equals(className)
+                    || "org.checkerframework.checker.nullness.qual.Nullable".equals(className)
```

Comment:
I would remove all the `.Nullable".equals()` above and replace them all with `className.endsWith(".Nullable")`.